app/src/main/java/ch/epfl/swissteam/services/CreatePostFragment.java

```diff
@@ -30,6 +34,10 @@
  */
 public class CreatePostFragment extends Fragment implements View.OnClickListener {
 
+    private SettingsDbHelper dbHelper_;
+    private String id_;
+    private boolean homeLocation;
```

Comment:
maybe choose another name, we could think that's a Location and not a boolean

Reply:
Sure ! I'll change to isHomeLocation